patroni/postgresql.py

```diff
@@ -73,14 +73,13 @@ def __init__(self, config):
 
         self._connection = None
         self._cursor_holder = None
-        self._need_rewind = False
         self._sysid = None
         self.replication_slots = []  # list of already existing replication slots
         self.retry = Retry(max_tries=-1, deadline=5, max_delay=1, retry_exceptions=PostgresConnectionException)
 
         self._state = 'stopped'
         self._state_lock = Lock()
-        self._role = 'replica'
+        self._role = 'replica' if os.path.exists(self.recovery_conf) else 'master'
```

Comment:
The same statement is used at  https://github.com/zalando/patroni/blob/feature/pg_rewind_unconditionally/patroni/postgresql.py#L350
Should we create method for that?